pom.xml

```diff
@@ -28,7 +28,7 @@ THE SOFTWARE.
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.520</version>
+    <version>1.609</version>
```

Comment:
This would be a good case for the parent pom (plugin-pom).  Just specify the jenkins.version as 1.609.